WordPress/src/main/java/org/wordpress/android/ui/prefs/categories/CategoriesListViewModel.kt

```diff
@@ -66,13 +71,18 @@ class CategoriesListViewModel @Inject constructor(
         }
     }
 
-    private fun fetchCategoriesFromNetwork(isInvokedFromInit: Boolean = false) {
+    private fun onRetryClicked() {
+        launch { fetchCategoriesFromNetwork(false) }
```

Comment:
Nitpick We can either remove explicit `false` here or include `isInvokedFromInit` param name for more clarity.

Reply:
@ashiagr Good observation here. I think removing explicit `false`would be better wdyt ?

Reply:
Yeah, it's not needed.

Reply:
@ashiagr 👍 Done in https://github.com/wordpress-mobile/WordPress-Android/pull/15603/commits/5400960d42d8fa2e157babc89a9431a2c8ead613